src/pcapi/repository/booking_queries.py

```diff
@@ -0,0 +1,24 @@
+from typing import List
+
+from pcapi.core.bookings.models import Booking
+from pcapi.core.offers.models import Offer
+from pcapi.core.offers.models import Stock
+from pcapi.core.users.models import User
+
+
+def find_bookings_by_beneficiary(user: User) -> List[Booking]:
+    return Booking.query.filter(Booking.user == user).all()
+
+
+def find_stocks_booked_by_beneficiary(user: User) -> List[Stock]:
```

Comment:
```suggestion
def find_offer_ids_booked_by_beneficiary(user: User) -> List[Stock]:
```

... puisqu'on renvoie des identifiants d'offre, pas des stocks.

Reply:
... et du coup, même question que plus haut : on rapporte aussi les réservations annulées ?

Reply:
Bonne remarque mais cette requête sert à mettre en avant les offres réservées par des comptes suspects, du coup peu importe si la réservation a été annulée ou non